Loop/Managers/DeviceDataManager.swift

```diff
@@ -609,58 +645,71 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate {
             carbRatioSchedule: carbRatioSchedule,
             insulinSensitivitySchedule: insulinSensitivitySchedule
         )
-
+        
         var idleListeningEnabled = true
-
+        
         if let pumpID = pumpID {
             let pumpState = PumpState(pumpID: pumpID)
-
+            
             if let timeZone = NSUserDefaults.standardUserDefaults().pumpTimeZone {
                 pumpState.timeZone = timeZone
             }
-
+            
             if let pumpModelNumber = NSUserDefaults.standardUserDefaults().pumpModelNumber {
                 if let model = PumpModel(rawValue: pumpModelNumber) {
                     pumpState.pumpModel = model
-
+                    
                     idleListeningEnabled = model.larger
                 }
             }
-
+            
             self.pumpState = pumpState
         }
-
+        
         rileyLinkManager = RileyLinkDeviceManager(
             pumpState: self.pumpState,
             autoConnectIDs: connectedPeripheralIDs
         )
         rileyLinkManager.idleListeningEnabled = idleListeningEnabled
-
+        
         if let  settings = NSBundle.mainBundle().remoteSettings,
```

Comment:
There's two spaces in `let  settings` so it aligns with the variables below it; if we move those indents up, we should move this back to one space: `let settings`.